packages/mobile/src/app/App.tsx

```diff
@@ -59,21 +58,13 @@ export class App extends React.Component {
     store.dispatch(openDeepLink(event.url))
   }
 
-  hideSplashScreen() {
-    SplashScreen.hide()
-  }
-
   render() {
     return (
       // @ts-ignore Apollo doesn't like the typings
       <ApolloProvider client={apolloClient}>
         <Provider store={store}>
           <SafeAreaProvider>
-            <PersistGate
-              onBeforeLift={this.hideSplashScreen}
```

Comment:
Can you test this an android too. I vaguely remember needing to add this onBeforeLift for android

Reply:
Yes works for Android too, I should have mentioned it ;)